### DISCUSSION TITLE
i want to add more column in setting tabel

### DISCUSSION THREAD

**Discussion by vahidalvandi** [spatie/laravel-settings#231] [Q&A]
I need to add more columns in the settings table and with the command  Quantify them

`$this->migrator->add('site.name', 'sample'); 
`

**Comment by rubenvanassche** [comment]
I'm not completely following what you're asking.

**Reply by mikemastercorp** [reply]
@vahidalvandi  what is stopping you from having more columns to your settings table? You just have to add them to your settings migration file and that is it. I for example have 4-5 in my migration, here is an example:

`<?php

use Spatie\LaravelSettings\Migrations\SettingsMigration;

return new class extends SettingsMigration
{
    public function up(): void
    {
        $this->migrator->add('general.site_name', 'Default Site Name'); // text input setting
        $this->migrator->add('general.site_email', 'default@example.com'); // text input with email validation
        $this->migrator->add('general.site_select', ''); // select dropdown input
        $this->migrator->add('general.site_is_checked', ''); // checkbox input
        $this->migrator->add('general.site_is_toggled', ''); // toggle input
        $this->migrator->add('general.site_checkboxlist', ''); // checkbox list input
        $this->migrator->add('general.site_radio', ''); // radio options input
        $this->migrator->add('general.meta_title', 'Default Meta Title'); // text input on another tab
        $this->migrator->add('general.meta_description', 'Default Meta Description'); // demo text input for the second tab
    }
};`

Of course this is just an example, but feel free to add as many as you like or just copy/paste my code and modify as required + add if any extra lines are needed. Hope that answers your question (if I understood correctly the issue...)

**Comment by ariesdg** [comment]
Hi, were you able to resolve this?

I already have existing setting in production, i want to additional migration to existing group.

**Reply by ariesdg** [reply]
Nevermind, just create another migration using make:settings-migration artisan command

**Reply by mikemastercorp** [reply]
@ariesdg 

You can always correct your existing migration (if not in PROD) but since you mentioned your app is already in PROD, you did it correctly by adding a new migration and importing only the newly required setting columns to your existing table.
